model/cluster_message.go

```diff
@@ -42,6 +42,7 @@ const (
 	CLUSTER_EVENT_REMOVE_PLUGIN                                     = "remove_plugin"
 	CLUSTER_EVENT_INVALIDATE_CACHE_FOR_TERMS_OF_SERVICE             = "inv_terms_of_service"
 	CLUSTER_EVENT_BUSY_STATE_CHANGED                                = "busy_state_change"
+	CLUSTER_EVENT_SYNC_SHARED_CHANNEL                               = "sync_shared_channel"
```

Comment:
i couldn't find a better name, so open to suggestions 😄 . Since this action is to signal the shared channel service to start syncing

Reply:
Name looks fine, but we may not need a new event type.  See comment above.